proto/fairyring/pep/request_id.proto

```diff
@@ -1,10 +1,23 @@
 syntax = "proto3";
 package fairyring.pep;
 
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "fairyring/common/shared_types.proto";
+
 option go_package = "github.com/Fairblock/fairyring/x/pep/types";
 
 message RequestId {
   string creator = 1; 
   string reqId = 2;
 }
 
+message PrivateRequest {
+  string creator = 1;
+  string req_id = 2;
+  string pubkey = 3;
+  cosmos.base.v1beta1.Coin amount  = 4 [(gogoproto.nullable) = false];
```

Comment:
let's get rid of this. the fee should be set as a global param instead of having it be up to the user.

Reply:
done